assets/app/scripts/services/navigate.js

```diff
@@ -48,6 +48,13 @@ angular.module("openshiftConsole")
         $location.path("project/" + encodeURIComponent(projectName) + "/create/next").search("name", name);
       },
 
+      toPodsForDeployment: function(deployment) {
+        $location.url("/project/" + deployment.metadata.namespace + "/browse/pods");
```

Comment:
i think it depended on whether you were using $location.path or $location.url, one of them i think was finicky and the other wasn't, url must handle either cleanly using the `<base>`